lib/zold/node/front.rb

```diff
@@ -66,6 +66,7 @@ class Front < Sinatra::Base
       set :wallets, nil? # to be injected at node.rb
       set :remotes, nil? # to be injected at node.rb
       set :copies, nil? # to be injected at node.rb
+      set :node_alias, nil? # to be injected at node.rb
```

Comment:
@emilianodellacasa just `alias` is not descriptive enough? https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@yegor256 The fact is that _alias_ is a reserved word for Ruby and it is not accepted, I tried again now and it gives a lot of syntax errors